src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -995,6 +996,11 @@ abstract class ManagedStrategy(
             return detectedFirstTime || detectedEarly
         }
 
+        fun onActorStart(iThread: Int) {
```

Comment:
Please add a comment explaining what this function does and why this logic is important